packages/adaptivecards-tools-sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft/adaptivecards-tools",
-  "version": "0.1.6",
+  "version": "1.0.0-rc",
```

Comment:
> please help update the adaptive-card ver in sdk devDeps to 1.0.0-rc

@eriolchan @yukun-dong I've updated this version because @microsoft/teamsfx depends on this package.